src/app/components/dashboard-company-selectbox/dashboard-company-selectbox.less

```diff
@@ -11,6 +11,7 @@ dashboard-company-selectbox {
     padding: @dashboard-cpy-select-padding;
     .border-bottom-radius(@dashboard-cpy-select-border-radius);
     z-index: 10;
+    border-radius: 0px 0px 0px 10px;
```

Comment:
This is overriding the `.border-bottom-radius(@dashboard-cpy-select-border-radius);` on  line 12